services/nginz/third_party/nginx-zauth-module/zauth_module.c

```diff
@@ -495,21 +497,29 @@ static ngx_int_t zauth_parse_request (ngx_http_request_t * r) {
 }
 
 static ZauthResult token_from_header (ngx_str_t const * hdr, ZauthToken ** t) {
-        if (strncmp((char const *) hdr->data, "Bearer ", 7) == 0) {
-                return zauth_token_parse(&hdr->data[7], hdr->len - 7, t);
+        char const * bearer = "Bearer ";
```

Comment:
If you define your strings as `char[]`, then you can use `sizeof` instead of hardcoding the length.